src/main/frontend/filtering.cljs

```diff
@@ -0,0 +1,43 @@
+(ns frontend.filtering
+  (:require [frontend.util :as util]
+            [frontend.state :as state]
+            [frontend.db :as db]))
+
+(defn get-block-references
```

Comment:
It's better to use `:block/ref-pages` here because the block body might have other links.